app/containers/IOSKeyboardAvoidingView.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { KeyboardAvoidingView, NativeModules, StatusBarIOS } from 'react-native';
+import PropTypes from 'prop-types';
+
+const { StatusBarManager } = NativeModules;
+
+const defaultKeyboardVerticalOffset = 44;
+
+export default class IOSKeyboardAvoidingView extends Component {
+	static propTypes = {
+		style: PropTypes.object,
+		children: PropTypes.oneOfType([
+			PropTypes.arrayOf(PropTypes.node),
+			PropTypes.node
+		]).isRequired
+	};
+
+	state = { statusBarHeight: 0 };
+
+	componentDidMount() {
+		StatusBarManager.getHeight((statusBarFrameData) => {
+			this.setState({ statusBarHeight: statusBarFrameData.height });
+		});
+		this.statusBarListener = StatusBarIOS.addListener('statusBarFrameWillChange', (statusBarData) => {
+			this.setState({ statusBarHeight: statusBarData.frame.height });
+		});
+	}
+
+	componentWillUnmount() {
+		this.statusBarListener.remove();
+	}
+
+	render() {
+		const { style, children } = this.props;
+		const { statusBarHeight } = this.state;
+
+		return (
+			<KeyboardAvoidingView
```

Comment:
I didn't like the experience.
I know you removed because I told you so, but let's stick with https://github.com/wix/react-native-keyboard-tracking-view.

Just to summarize:
- iOS: will use https://github.com/wix/react-native-keyboard-tracking-view
- Android: will use https://github.com/wix/react-native-keyboard-input and https://github.com/wix/react-native-keyboard-tracking-view